src/background/zcl_abapgit_background.clas.abap

```diff
@@ -135,14 +135,14 @@ CLASS ZCL_ABAPGIT_BACKGROUND IMPLEMENTATION.
 
   METHOD run.
 
-    DATA: lo_per         TYPE REF TO zcl_abapgit_persist_background,
-          li_repo        TYPE REF TO zif_abapgit_repo,
-          li_repo_online TYPE REF TO zif_abapgit_repo_online,
-          lt_list        TYPE zcl_abapgit_persist_background=>ty_background_keys,
-          li_background  TYPE REF TO zif_abapgit_background,
-          li_log         TYPE REF TO zif_abapgit_log,
-          lx_error       TYPE REF TO zcx_abapgit_exception,
-          lv_repo_name   TYPE string.
+    DATA:
+      li_repo        TYPE REF TO zif_abapgit_repo,
+      li_repo_online TYPE REF TO zif_abapgit_repo_online,
+      lt_list        TYPE zif_abapgit_persist_background=>ty_background_keys,
+      li_background  TYPE REF TO zif_abapgit_background,
+      li_log         TYPE REF TO zif_abapgit_log,
+      lx_error       TYPE REF TO zcx_abapgit_exception,
+      lv_repo_name   TYPE string.
```

Comment:
one line here that is not a whitespace change?

Reply:
i don't make pretty print